src/Data/Managers/V2/helpers/useHandleErrors/useHandleErrors.ts

```diff
@@ -6,37 +6,15 @@ import { useContext } from "react";
 import { DependencyContext } from "@/UI";
 
 export const useHandleErrors = () => {
```

Comment:
Is it really necessary now to have a hook for this?

Reply:
removed